Update whatis entry for manual page to something more useful.
Index: dcfldd-1.3.4.1/dcfldd.1
===================================================================
--- dcfldd-1.3.4.1.orig/dcfldd.1	2012-05-12 23:37:00.033037122 +0100
+++ dcfldd-1.3.4.1/dcfldd.1	2012-05-12 23:40:26.159509098 +0100
@@ -1,7 +1,7 @@
 .\" DO NOT MODIFY THIS FILE!  It was generated by help2man 1.35.
 .TH DCFLDD "1" "February 2006" "dcfldd (dcfldd) 1.3.4" "User Commands"
 .SH NAME
-dcfldd \- manual page for dcfldd (dcfldd) 1.3.4
+dcfldd \- enhanced version of dd for forensics and security
 .SH SYNOPSIS
 .B dcfldd
 [\fIOPTION\fR]...
